src/coreComponents/mesh/ElementRegionManager.cpp

```diff
@@ -268,7 +269,8 @@ void ElementRegionManager::buildSets( NodeManager const & nodeManager )
     {
       arrayView1d< bool const > const nodeInCurSet = nodeInSet[setName];
 
-      SortedArray< localIndex > & targetSet = elementSets.registerWrapper< SortedArray< localIndex > >( setName ).reference();
+      SortedArray< localIndex > & targetSet = elementSets.registerWrapper< SortedArray< localIndex > >( setName, true ).reference();
+      targetSet.clear();
```

Comment:
This `targetSet.clear()` is different from the previous behavior, but seems likely to be what we want.